eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -238,7 +238,11 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
               }
           }
           // no need to go OFFLINE, we can directly switch to CLOSING
-          goto(CLOSING) using closing
+          if (closing.waitingSinceBlock > 1500000) {
+            goto(CLOSING) using closing.copy(waitingSinceBlock = nodeParams.currentBlockHeight)
```

Comment:
If we don't update the db we'll push back forever:
```suggestion
            goto(CLOSING) using closing.copy(waitingSinceBlock = nodeParams.currentBlockHeight) storing()
```

NB: we don't seem to be using `waitingSinceBlock` for anything in state `CLOSING`?

Reply:
We are, because we do a `GetTxWithMeta` if the funding tx is unconfirmed (and it allows us to move to the `CLOSED` state).

Reply:
Right. The meaning of `waitingSinceBlock ` is really ambiguous though, because we are naming it and setting it as "waiting in `CLOSING` since" but we are actually using it as "waiting for funding tx to confirm since". Don't you find this odd? I know it's unrelated to this PR.

For example here we could be using `waitingSinceBlock = waitForFundingConfirmed.waitingSinceBlock`
https://github.com/ACINQ/eclair/blob/48ac6b522dd9ee0b1fae8a143925e54f2e55f1a1/eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala#L2190

Reply:
> The meaning of waitingSinceBlock is really ambiguous though, because we are naming it and setting it as "waiting in CLOSING since" but we are actually using it as "waiting for funding tx to confirm since". Don't you find this odd?

Yes I agree, I can dive more into this and clean that up.

Reply:
I think it could be worth it.

Reply:
I added a comment in https://github.com/ACINQ/eclair/pull/1692/commits/4019b6693dd52e5537b0c1c7b9db3cde7fa08942

I think it's useful to have `Closing.waitingSinceBlock` really be the block when the *closing* was initiated (and not the waiting-for-confirmed-since-block): it can help check how much time we spend in closing (and draw nice graphs for later analysis).

The only drawback is that it "resets" the funding timeout counter (we allow more time for the funder) but I think that's ok.